online/decoder_maindaq/MainDaqParser.cc

```diff
@@ -805,95 +805,93 @@ int MainDaqParser::ProcessBoardTriggerCount (int* words, int j)
  */
 int MainDaqParser::ProcessBoardFeeQIE (int* words, int idx)
 {
-    if (dec_par.runID < 22400) Abort("feeQIE does not support run < 22400.");
```

Comment:
The only meaningful change in this file is the removal of this line.  This run-number condition is valid only for E906 data.

The other changes in this file are just of the size of the line indent.